ci/collect-garbage.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ../shell.nix
+# shellcheck shell=bash
+
+set -eux
+
+scriptroot="$(dirname "$(realpath "$0")")"
+NRHOSTS="$(ls "$scriptroot"/../morph-network/machines/*.expr.nix | wc -l)"
+
+step() {
+  host="$1"
+
+  cat <<EOF
+  - label: "nix-collect-garbage"
+    concurrency_group: ofborg-infrastructure-deploy
```

Comment:
```suggestion
    concurrency_group: ofborg-infrastructure-gc
```

Reply:
Only reason I left it as `-deploy` is so that it would block the dry activate etc. steps. Maybe I should just `depends_on: nix-collect-garbage` for the dry activate step?

Reply:
:+1: makes sense